CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+
+## Forest v0.3.0 (unreleased)
+
+Notable updates:
+* Support nv15 entirely through the FVM.
+* Resolve two security concerns by removing legacy code (RUSTSEC-2020-0071 and RUSTSEC-2021-0130).
+* Fixed SSL certificates in docker image.
```

Comment:
I'd say just `fixed Docker image and released it to GH container registry` as the previous one was in a state where it was not even building nor it was published anywhere. What do you think?